src/components/controls/measurementsOptions.js

```diff
@@ -112,9 +112,12 @@ const MeasurementsOptions = () => {
       />
       <Toggle
         // Only display threshold toggle if the collection has a valid threshold
-        display={typeof collection.threshold === "number"}
+        display={
+          Array.isArray(collection.thresholds) &&
+          collection.thresholds.some((threshold) => typeof threshold === "number")
+        }
         on={showThreshold}
-        label="Show measurement threshold"
+        label="Show measurement threshold(s)"
```

Comment:
_non-blocking_

This diff made me realize two things.

Labels like this should go thru our i18n framework (i18next) so they can be translated in the future.

```jsx
label={t("Show measurement threshold(s)")}
```

Once they do, we can avoid the ugly "(s)" by using the [standard pluralization/number agreement features](https://www.i18next.com/translation-function/plurals), e.g. something like:

```jsx
label={t("Show measurement threshold", {count: collection.thresholds.length})}
```

with corresponding translations, e.g. for `en` something like:

```json
{
  "Show measurement threshold_one": "Show measurement threshold",
  "Show measurement threshold_others": "Show measurement thresholds",
}
```

(I realize these are out of scope for this PR.)

Reply:
Oh yeah, I'm totally missing the translation feature for all measurements options...

Very cool that i18next supports plurals formatting like this! 
I decided that I'm going to punt this for now as I was reading the i18next docs since we are still using i18next v19.9.2 and I haven't dug into which JSON format we are using:
> Starting with i18next>=21.3.0 you can use the built-in formatting functions based on the Intl API

> You may need to polyfill the Intl.PluralRules API, in case it is not available it will fallback to the i18next JSON format v3 plural handling.